prisma/seed.ts

```
@@ -491,6 +539,19 @@ async function main() {
 
     console.log(`Creating seed data for real user: ${realUser.firstName} ${realUser.lastName}`);
 
+    // Set founder role and create invite code for real user
+    await db.user.update({ where: { id: realUser.id }, data: { role: "founder" } });
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'data' should be before 'where'.

(sort-keys-fix/sort-keys-fix)